src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5993,6 +5993,7 @@ def test_create_image_version_with_region_cvm_encryption(self, resource_group, r
         self.cmd('sig image-version delete -g {rg} --gallery-name {gallery} --gallery-image-definition {image} --gallery-image-version {version}')
         time.sleep(60)  # service end latency
         self.cmd('sig image-definition delete -g {rg} --gallery-name {gallery} --gallery-image-definition {image}')
+        time.sleep(60)
```

Comment:
An extra sleep is added to fix https://github.com/Azure/azure-cli/pull/26577#issuecomment-1680001533 according to https://github.com/Azure/azure-cli/pull/26577#issuecomment-1680051185.

`time.sleep` is patched during playback mode, so there is no need to add `if self.is_live:` check:

https://github.com/Azure/azure-cli/blob/30216d0268f3491b8af2aa060d90a9e1f94f9b3c/src/azure-cli-testsdk/azure/cli/testsdk/scenario_tests/patches.py#L9-L13